nifpga/bitfile.py

```diff
@@ -381,7 +381,12 @@ class _FXP(_BaseType):
     def __init__(self, name, type_xml):
         super(_FXP, self).__init__(name)
         self._datatype = DataType.Fxp
-        self._signed = True if type_xml.find("Signed").text.lower() == 'true' else False
+        signed_tag = type_xml.find("Signed")
+        if signed_tag is None:
+            raise UnsupportedTypeError("Unsupported FXP type encountered. This bitfile "
```

Comment:
When we raise an exception does the object not get created? I wouldn't want the session to end up with broken registers.

Reply:
If a known exception is thrown while parsing the type, the Register or FIFO currently being parsed is omitted.  This same mechanism is used to avoid CFXP registers until we add support for them.
https://github.com/ni/nifpga-python/blob/master/nifpga/bitfile.py#L34